e2e/specs/clinical-forms.spec.ts

```diff
@@ -36,7 +36,6 @@ test('Fill a clinical form', async ({ page, api }) => {
     await expect(headerRow).toContainText(/form name \(a-z\)/i);
     await expect(headerRow).toContainText(/last completed/i);
 
-    await expect(chartPage.page.getByRole('cell', { name: /ampath_poc_adult_return_visit_form/i })).toBeVisible();
```

Comment:
@chimanwadike is this test failing in GitHub Actions?

Reply:
@kdaud I wouldn't know if it will fail on Github Action. (since Github Action execute only after a PR approval review)

I decided to run a full e2e on my local before working on my first task and then found out this error.
I suppose fresh installation doesn't come with Ampath Adult Return Visit.
Depends on what instance Github Action is looking at, I basically cloned and ran this same repo.
 
PS: I'm new to the community pardon if I'm missed communicating well

Reply:
@kdaud Yes this is failing e2e test.
And removal of `ampath_poc_adult_return_visit_form` fixes the issue as @chimanwadike proposed.

Reply:
@chimanwadike @Vijaykv5 the test passes on GitHub Actions running on a local instance but fails on dev3 server.